frontend/src/components/AnnotationView/TagDataset.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div class="dataset-container">
+    <span v-if="label && !linkout" class="dataset-label">{{ label }}</span>
+    <a v-else-if="label && linkout" :href="linkout" class="dataset-label" target="_blank" rel="noreferrer noopener">
+      {{ label }}
+      <font-awesome-icon icon="up-right-from-square" size="2xs" />
+    </a>
+    <div v-if="!isDataUnavailable" class="tags-container">
+      <div class="tag" v-for="(item, index) in content" :key="index">
+        {{ item }}
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'tag-dataset',
+  props: {
+    label: {
+      type: String,
+    },
+    linkout: {
+      type: String,
+      required: false,
+    },
+    value: {
+      type: [String, Array],
+    },
+    delimeter: {
+      type: String,
+      default: '; ',
+    },
+  },
+  computed: {
+    isDataUnavailable() {
+      return this.value == '.' || this.value == 'null' || this.value == null;
+    },
+    dataAvailabilityColour() {
+      return this.isDataUnavailable ?
+        'var(--rosalution-grey-300)' :
+        this.linkout ? 'var(--rosalution-purple-300)' :
+          'var(--rosalution-black)';
+    },
+    content() {
+      if (typeof (this.value) == 'object') {
+        return this.value;
+      }
+
+      return this.value.split(this.delimeter);
+    },
+  },
+};
+</script>
+
+<style scoped>
+.dataset-container {
+  display: flex;
+  padding: var(--p-1);
+  line-height: 24px;
+}
+
+.dataset-label {
+  flex: 0 0 130px;
+  font-weight: 600;
+  color: v-bind(dataAvailabilityColour)
+}
+
+.tags-container {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 10px 5px;
+}
+
+.tag {
+  padding: 5px 10px;
```

Comment:
Using the design system padding is required to keep the padding consistent among the entire application.  This also leaves the padding consistent in REM when we are handling different screen resolutions and zoom percentages.  

Wherever applicable, update the PR using the CSS custom properties (variables) to match their usage in other components of the application and as you have been doing previously.

Reply:
I did play around with the variables and used slightly different sizing.   For this attribute, `var(--p-1) var(--p-8)` looks good and makes it tighter.